src/app/features/container/utils/get-popup-header.ts

```diff
@@ -4,21 +4,41 @@
 import { RouteUrls } from '@shared/route-urls';
 
 export function showAccountInfo(pathname: RouteUrls) {
-  return (
-    pathname === RouteUrls.TransactionRequest ||
-    pathname === RouteUrls.ProfileUpdateRequest ||
-    pathname === RouteUrls.PsbtRequest
-  );
+  switch (pathname) {
```

Comment:
We should clean this up when working on `Approval UX` and implement a popup clear variant pattern rather than rely on routing